File: src/main/java/com/patres/timetable/config/ApplicationProperties.java
package com.patres.timetable.config;

import org.springframework.boot.context.properties.ConfigurationProperties;

/**
 * Property specific to JHipster.
 * <p>
 * Property are configured in the application.yml file.
 */
@ConfigurationProperties(prefix = "application", ignoreUnknownFields = false)
public class ApplicationProperties {

}

File: src/main/kotlin/com/patres/timetable/config/ApplicationProperties.kt
package com.patres.timetable.config

import org.springframework.boot.context.properties.ConfigurationProperties

/**
 * Property specific to JHipster.
 *
 *
 * Property are configured in the application.yml file.
 */
@ConfigurationProperties(prefix = "application", ignoreUnknownFields = false)
class ApplicationProperties
